core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -160,12 +164,14 @@ sealed trait ZIO[-R, +E, +A] extends Serializable with ZIOPlatformSpecific[R, E,
   /**
    * Operator alias for `orElse`.
    */
+  // TODO: This usually the monoid operator
   final def <>[R1 <: R, E2, A1 >: A](that: => ZIO[R1, E2, A1])(implicit ev: CanFail[E]): ZIO[R1, E2, A1] =
```

Comment:
I think this one is also sometimes actually used. It also actually is an associative binary operator and the most information preserving one you can define.